TFT/src/User/my_misc.c

```diff
@@ -145,7 +145,7 @@ const char *stripHead(const char *str)
 {
   // example: " :/test/cap2.gcode\n" -> "test/cap2.gcode\n"
 
-  while (*str == ':' || *str == '/' || *str == ' ')
+  while (*str == ' ' || *str == ':' || *str == '/')
```

Comment:
In my opinion it's best to put the most probably frequent conditionals first to avoid unnecessary checks

Reply:
this function is typically used for such kind of strings:
`M20 SD:      /my_filder/my_file.gco`
where the keyword `M20 SD:` was previously removed so the string passed to this function is `      /my_filder/my_file.gco`. So yes, ` ` is the only character that will be repeated and the first one that will appear. Then one `/` and finally `:` (that will probably never be present). Also this function is called when the system is on idle (not printing).